model/clusters_mgmt/v1/log_forwarder_group_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Represents a log forwarder group.
+struct LogForwarderGroup {
+    // The name of the log forwarder group.
+    Name String
```

Comment:
@willkutler I'd make this `id`. `Name` can be reserved for a future human-readable name (not in scope at this time) along with description. See how we do this for `/api/cluster_mgmt/v1/versions`